NBISweden/sda-cli
# split integration tests

**Related issue(s) and PR(s)**  
This PR closes #403 .


**Description**
The file with integration tests is split into:
```
10_encrypt_decrypt.sh
20_upload.sh
30_list.sh
40_download.sh 
```

**How to test**
Make sure the tests in github actions runs, and that you can run all tests locally, eg:
```
.github/integration/setup/setup.sh  # some manual intervention might be needed here, at least for me, but no changes from before
.github/integration/tests/10_upload.sh
.github/integration/tests/20_encrypt_decrypt.sh
.github/integration/tests/30_list.sh
.github/integration/tests/40_download.sh
```
